sr_port/cache_stats.c

```diff
@@ -21,13 +21,15 @@ GBLREF	hash_table_objcode	cache_table;
 
 void cache_stats(void)
 {
-	int		total_attempts, ace;
+	long long	total_attempts, ace, cache_hits_ll, cache_fails_ll;
```

Comment:
Coding convention thing. "gtm_uint64_t" is preferable to long long. Also, would be good to remove cache_hits_ll and cache_fails_ll and instead make the global variable cache_hits and cache_fails of type "gtm_uint64_t". That keeps the change simpler and enhances YottaDB to work fine if the indirection cache hits counter goes > 4Gb (possible in long-running server processes).

Reply:
Changing cache_hits and cache_fails to gtm_uint64_t globally affects more parts of the program - as discussed, I'm instead casting the hit ratio calculation from int to gtm_uint64_t.